airdrop/infrastructure/repositories/airdrop_repository.py

```diff
@@ -151,13 +151,22 @@ def register_airdrop_window_timeline(self, airdrop_window_id, title, description
             airdrop_window_id=airdrop_window_id, title=title, description=description, date=date)
         return self.add(airdrop_window_timeline)
 
+    def get_contract_address(self, airdrop_id):
+        airdrop = self.session.query(Airdrop).filter_by(
+            airdrop_id=airdrop_id).first()
+
+        if airdrop is None:
+            raise Exception('Airdrop not found')
+
+        return airdrop.contract_address
+
     def get_token_address(self, airdrop_id):
         airdrop = self.session.query(Airdrop).filter_by(id=airdrop_id).first()
 
         if airdrop is None:
             raise Exception("Airdrop not found")
 
-        return airdrop.contract_address
+        return airdrop.address
```

Comment:
May be for future , address needs to be prefixed by what they mean

Reply:
It will be addressed in a separate PR @anandrgitnirman